docker-compose.yml

```diff
@@ -112,7 +111,6 @@ services:
     restart: always
     environment:
       FIDES__CONFIG_PATH: ${FIDES__CONFIG_PATH:-/fides/.fides/fides.toml}
-      FIDES__TEST_MODE: "True"
```

Comment:
Removing setting `FIDES__TEST_MODE` in the `docker-compose.yml`.  In the spirit of how fidesops worked, this variable will only be set by pytest.